src/components/Card/index.tsx

```diff
@@ -1,9 +1,8 @@
 import React from 'react'
 import styled from 'styled-components'
-import { CardProps, Text } from 'rebass'
-import { Box } from 'rebass/styled-components'
+import { Text } from '@pancakeswap-libs/uikit'
 
-const Card = styled(Box)<{ padding?: string; border?: string; borderRadius?: string }>`
+const Card = styled.div<any>`
```

Comment:
This is the start of getting rid of Rebass

Reply:
you can removed the `any`, it will implicitly get the styled type